Rollbar-Teams-Membership/test/integ-template.yml

```diff
@@ -0,0 +1,12 @@
+Resources:
+  MySampleTeam:
+    Type: Rollbar::Teams::Team
```

Comment:
Will using a type that is part of the same pipeline cause issues? There is no guarantee that this type would exist in the region when this runs I believe.